distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -918,6 +919,72 @@ type Table
         result = Table.new new_columns
         problem_builder.attach_problems_after on_problems result
 
+    ## Splits a column of text into a set of new columns.
+       The original column will be removed from the table.
+       The new columns will be named with the name of the input column with a
+       incrementing number after.
+
+       Arguments:
+       - column: The name or index of the column to split the text of.
+       - delimiter: The term or terms used to split the text.
+       - column_count: The number of columns to split to.
+         If `Nothing` then columns will be added to fit all data.
+         If the data exceeds the `column_count`, a `Column_Count_Exceeded` error
+         will follow the `on_problems` behavior.
+       - on_problems: Specifies the behavior when a problem occurs.
+    split_to_columns : Text | Integer -> Text -> Integer | Nothing -> Problem_Behavior -> Table
+    split_to_columns self column delimiter="," column_count=Nothing on_problems=Report_Error =
+        Split_Tokenize.split_to_columns self column delimiter column_count on_problems
```

Comment:
The docstrings on `Standard.Table` `Table` should be the same as on `Standard.Database` `Table`. Please ensure they are consistent. I guess we may want to add a test which checks this, like Jaroslav did in #6218 PR.